app/models/registry.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env ruby
```

Comment:
Yeah, I thought that as well.

Thanks for the catch, I forgot to remove these lines.